x-pack/plugins/infra/public/containers/metrics_source/source_errors.ts

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+/* eslint-disable max-classes-per-file */
+
+import { HttpFetchError } from '@kbn/core-http-browser-internal/src/http_fetch_error';
+import { i18n } from '@kbn/i18n';
+
+const NO_SUCH_REMOTE_CLUSTER = 'no_such_remote_cluster_exception';
+
+const defaultTitle = i18n.translate('xpack.infra.sourceConfiguration.failedToFetchSource', {
+  defaultMessage: 'Failed source request.',
```

Comment:
Maybe we should word this in more user friendly terms, like "Failed to load source configuration"?